.storybook/preview-head.html

```diff
@@ -1,48 +1,8 @@
 <link
-  rel="preload"
-  href="/static/media/Lato-Black.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-BlackItalic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Bold.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-BoldItalic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Italic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Regular.woff2"
+  rel="preload stylesheet"
+  href="https://fonts.sage.com/sage.css"
   as="font"
```

Comment:
```suggestion
  as="style"
```
This is now a css file rather than a font file so I think this need to be changed